### DISCUSSION TITLE
🚀 Nextra Docs Structure Generator

### DISCUSSION THREAD

**Discussion by dharanigowthamsampath** [shuding/nextra#3777] [Show and tell]
Hi everyone,

I've been using Nextra for almost a month now, and I'm trying to create documentation for all languages under a single domain (developerdocs.in). You can check out my project here: [Nextra File Generator](https://dharanigowthamsampath.github.io/Nextra-File-Generator).

I really like Nextra, but creating the file and folder structure for lengthy documentation with `meta.js` has been quite challenging. While I can generate syllabi using GPT, generating the actual files, folders, and meta files has been tough.

To address this, I created a simple app that takes JSON as input and generates the pages folder and meta files based on that JSON. I believe this tool could be useful for many of you. I'm planning to add more features in the future.

Looking forward to your feedback!

--- 

Feel free to modify any part of it to better fit your style or add any additional information you think is necessary!

**Comment by thedaviddias** [comment]
Super cool tool! Nice work @dharanigowthamsampath!
